smart-on-fhir-app/static/js/script.js

```diff
@@ -0,0 +1,9 @@
+// function to verify the app is deploy
+function loadStatus() {
+  let time = new Date();
+  let url = window.location.href;
+
+  let ackJSON = { url: url, time: time };
+
+  document.body.innerHTML = `<pre>` + JSON.stringify(ackJSON) + `</pre>`;
```

Comment:
put the file name as `static/js/utils.js` as I have told @safurazaheer in this [comment](https://github.com/sid22/smart-on-fhir-demo/pull/17#pullrequestreview-406780246).